OpenSim/Common/Component.cpp

```diff
@@ -615,86 +615,27 @@ void Component::setParent(const Component& parent)
 
 std::string Component::getFullPathName() const
```

Comment:
I am fine with either `getString()` or `toString()`. Perhaps `toString()` makes a little more sense, since `get<...>()` is usually used for accessing member variables, whereas `toString()` indicates that a "type conversion" is happening.

To clarify: @aseth1, you are saying to *not* add `ComponentPath ComponentPath::getRelativePathName()` for this PR, correct? I think that is fine. We can add them later.